ledger-test-tool/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/Features.scala

```diff
@@ -21,6 +21,7 @@ final case class Features(
     explicitDisclosure: Boolean = false,
     userAndPartyLocalMetadataExtensions: Boolean = false,
     acsActiveAtOffsetFeature: Boolean = false,
+    templateFilters: Boolean = false,
```

Comment:
Why do you need this flag? Can't you rely on old LAPITT versions to test the deprecated behavior?

Reply:
I want this new tool to test both old and new cantons and adopt its machinery to both

Reply:
So you found that old test machinery had less coverage, you mean. Fine by me then